OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -1077,7 +1097,10 @@ public void shouldNotSetAlertnessFieldsOnLowPriority() {
    @Test
    public void shouldAddDefaultButtonToAlertDialog() throws Exception {
       OneSignal.setInFocusDisplaying(OneSignal.OSInFocusDisplayOption.InAppAlert);
-      OneSignal.startInit(blankActivity).init();
+//      OneSignal.startInit(blankActivity).init();
```

Comment:
Delete commented out line

Reply:
Left6 some comments so we knew where it came from but will delete now